Jenkinsfile

```diff
@@ -25,10 +25,11 @@ def buildImages = { ->
       imageNamePy2 = "${imageNameBase}:py2-${gitCommit()}"
       imageNamePy3 = "${imageNameBase}:py3-${gitCommit()}"
       imageDindSSH = "${imageNameBase}:sshdind-${gitCommit()}"
-
-      buildImage(imageDindSSH, "-f tests/Dockerfile-ssh-dind .", "")
-      buildImage(imageNamePy2, "-f tests/Dockerfile --build-arg PYTHON_VERSION=2.7 .", "py2.7")
-      buildImage(imageNamePy3, "-f tests/Dockerfile --build-arg PYTHON_VERSION=3.7 .", "py3.7")
+      withDockerRegistry(credentialsId:'dockerbuildbot-index.docker.io') {
```

Comment:
can we use an unprivileged account, and no access to any private images in the docker namespaces? ideally a PAT

Reply:
We just compared it to the docker/compose repo and used the same here. 😅 
And this pipeline also pushes the images. 🤔

Reply:
Do we have a better hub account that we could use?

Reply:
if needed, we could create one perhaps

Reply:
If it's currently pushing; what credentials does it use for that?

(sorry for brevity; on my phone 😂)

Reply:
I've updated the Jenkins secret to use a PAT instead of the real password.